Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.cpp

```diff
@@ -288,39 +288,41 @@ ExceptionOr<void> WebCodecsAudioEncoder::encode(Ref<WebCodecsAudioData>&& frame)
     }
     ASSERT(!frame->isDetached());
 
-    if (m_state != WebCodecsCodecState::Configured)
+    if (state() != WebCodecsCodecState::Configured)
         return Exception { ExceptionCode::InvalidStateError, "AudioEncoder is not configured"_s };
 
-    ++m_encodeQueueSize;
```

Comment:
Again, we should increment here I think.

Reply:
I like that that handling of the message control queue is handled in the bast class; maybe I could use a different name?

incrementing is fine, the issue is having to decrement only if the codec isn't saturated.